Add number platform to Husqvarna Automower (#32197)
home-assistant/home-assistant.github.io: source/_integrations/husqvarna_automower.markdown

- Pause mowing
- Park until next schedule

### Number (if available)

> The integration will create a number entity for changing the cutting height of the mower. This entity is disabled by the default. You have to enable it manually, because it can't be detected with the API if the mower has the capabilities to change the cutting height remotely. Consult the handbook of the mower before you enable this function. Also it's possible, that you can use this entity only as sensor and not actively change the cutting height.

Reviewer: ```suggestion
The integration will create a number entity for changing the cutting height of the mower. This entity is disabled by default. You have to enable it manually because it can't be detected with the API if the mower has the capability to change the cutting height remotely. Before enabling this function, consult the handbook of the mower. It's possible that you can use this entity only as sensor but cannot actively change the cutting height.
```
